sdk/schemaregistry/schema-registry/samples/v1-beta/typescript/sample.env

```diff
@@ -0,0 +1,13 @@
+SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE=<FULLY QUALIFIED NAMESPACE FOR AN AVRO GROUP>
+SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE=<FULLY QUALIFIED NAMESPACE FOR A JSON GROUP>
+SCHEMAREGISTRY_AVRO_FULLY_QUALIFIED_NAMESPACE=<FULLY QUALIFIED NAMESPACE FOR A CUSTOM GROUP>
```

Comment:
```suggestion
SCHEMAREGISTRY_JSON_FULLY_QUALIFIED_NAMESPACE=<FULLY QUALIFIED NAMESPACE FOR A JSON GROUP>
SCHEMAREGISTRY_CUSTOM_FULLY_QUALIFIED_NAMESPACE=<FULLY QUALIFIED NAMESPACE FOR A CUSTOM GROUP>
```

Reply:
done in https://github.com/Azure/azure-sdk-for-js/pull/23575/commits/f76bcd8e90f9573bebeb8c20c7c75b719596330f